docs/tutorials/content.md

```diff
@@ -0,0 +1,155 @@
+- Example of the creation of globe view and planar view can be found in tutorial (with links to tutorial).
+
+### 3 - The data supported by iTowns
+
+- ITowns can display data from a wide range of sources, such as servers using WMS, WMTS, WFS, TMS protocols or user given files.
+  
+- The format of the displayable data are multiple too : vector tiled resources from Mapbox, Potree 3D point cloud file, oriented image file, GeoJSON, KML or GPX file...
+- ITowns offers the possibility to add new sources.
+
+> **_PART TO VALIDATE LATER_**
+>### 4 - From the raw data to the `View`
+>
+>- The process of displaying data is a three stages process.
+>- The data is first loaded from its source.
+>- It is then parsed into an iTowns utility format.
+>- Finally, the data is converted into a 3D object that can be displayed on the `View`.
+>- PRECISION 1 : If the data is a raster data, a 3D tile is created.
+>  This tile is a support object on which the raster content is applied.
+>  Example : picture of some raster images printed on a tile ? (something like the following)
+>
+>| ![image 1](images/basics-1.png) | ![image 2](images/basics-2.png) | ![image 3](images/basics-3.png)|
+>| --- | --- | --- |
+>
+>- PRECISION 2 : If the data is vector data, the parsed data can either be rasterized and then display as a raster data, or it can be converted into 3D objects that shall be displayed on the `View`.
+>  Example : picture of some vector data (like a polygon) printed on a tile ?
+
+### 5 - Data visualization support
+
+- As previously mentioned, data is displayed as layered data in iTowns (`Layer`, `ColorLayer`, `ElevationLayer`, `GeometryLayer`, `RasterLayer`).
+- A word on what each `Layer` allows to visualize (with potential image to illustrate layers).
+
+### 6 - Vector data processing
+
+- To display vector data, iTowns parses it to create instances of `Feature`.
+- A `Feature` holds a `type` property, which defines the geographic object it contains (points, lines, polygons).
+- The features are attached to layers.
+    - If the feature is to be rasterized, it is attached to a `ColorLayer`.
+    - If it is to be converted into 3d objects, it is attached to a `GeometryLayer`.
+- The appearance of the Layers and of the features it may contain can be modified with `Style`
+- `Style` holds several properties that define different aspects of a displayed feature :
+    - It can be applied to polygons (`fill` and `stroke`), to points (`point`) and to text labels (`text`).
+    - It allows changing a wide range of characteristics of the displayed vector data, such as their color, their transparency, their dimensions, labels...
+    - `Style` properties can be set from properties contained within the data. 
+      For instance, one could reference the `Style` property that define the content of a text label as one of the properties contained within a GeoJSON he wishes to display.
```

Comment:
re write to user cases.
* data vector to terrain stylized by object properties
* data vector to 3D object stylized by the user

Reply:
* data vector to 3D object on wrapped terrain.